### ISSUE TITLE
Release 1.0.0

### ISSUE THREAD

**Issue by dariocravero** [padrino/padrino-framework#975]
Placeholder to discuss release when its ready.

**Comment by namusyaka** [comment]
I'd like to start moving toward 1.0.0. @padrino/core-members

**Comment by nesquena** [comment]
I agree with you @namusyaka. At this point we might as well go to 1.0. We have the new site launched. Padrino is clearly fairly stable at this point.

**Comment by ujifgc** [comment]
There's some issues we can not resolve without changing API.

**Comment by namusyaka** [comment]
@ujifgc Could you enumerate the issues?

**Comment by ujifgc** [comment]
Not really. I'm just unhappy about current API and its inline documentation. It's too diverse. 1.0.0 should mean stable beautiful API.

**Comment by namusyaka** [comment]
We should discuss the beautiful API.
And I want to lay out a concrete road map.

**Comment by nesquena** [comment]
Yeah we can do preview releases with the breaking API changes or do one more major version before 1.0 to work out any kinks. What would you all suggest as the cornerstone breaking changes? 

I've thought in the past we might want to simplify the syntax for controller routing. I have a related issue. #747

**Comment by namusyaka** [comment]
I want to improve controller ang routing syntax, but i'd like to hear other opinions. With that in mind, we should list-up concreate milestones for releasing v1.0.0.

**Comment by nesquena** [comment]
This was once our working draft, obviously may not be totally relevant now: https://github.com/padrino/padrino-framework/wiki/Roadmap-for-1.0
